recipes/earth2observe/LICENSE.md

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
Again, please confirm that ou need this here. You might as well just pull it from the package.